### PR TITLE
✨ greet message enhancement

### LINKED ISSUE #119: Prompt to Check Inbox for Invite
## Expected Behavior
After a successful request user should get a prompt to **wait and check for email** as some users keep on requesting again.

New message `Thank You, Check your inbox for the invite`

## Current Behavior and Triggers
Currently, the user is getting await 

image

## Possible Solution
Here we can change the message to `Thank You, Check your inbox for the invite`
and `uk-spinner` to `uk-icon='icon: check; ratio: 2'`

image

## Your Environment
N/A

## Would you like to work on this issue?

Yes

### PR DESCRIPTION
**Pull request by AmanRaj1608**
Fixes #119 

### Description

image

### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Non Functional Requirement
- [x] Follows the code style of this project.
- [x] Tests Cover Changes
- [x] I have performed a self-review of my own code
- [x] All new and existing tests passed.
- [ ] Documentation

### CODE REVIEW

**Review by yashshah2820 — APPROVED** [review]

**Review by thepushkarp — APPROVED** [review]
Looks good!

File: public/client.js [hunk 1]
```diff
@@ -164,7 +164,7 @@ window.showToast = function () {
   } else {
     // eslint-disable-next-line
     UIkit.notification({
-      message: '<div uk-spinner></div> <span class=\'uk-text-small\'>Processing your request.</span>',
+      message: `<span uk-icon='icon: check; ratio: 1.5'></span> <span class=\'uk-text-small\'>Thank You, Check your inbox for the invite.</span>`,
```
Comment (aashutoshrathi) [inline public/client.js:167]: Should we use `uk-icon='icon: mail; ratio: 1.5'`?

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
Wouldn't 📫 icon look better?

File: public/client.js (hunk 1 above)
Comment (AmanRaj1608) [reply public/client.js:167]: Yeah it looks better now 

image

**Review by aashutoshrathi — APPROVED** [review]
LGTM

**Pull request merged**